From 7467a867b2fbb81f1150e129861cdb5965b50927 Mon Sep 17 00:00:00 2001
From: Hu Butui <hot123tea123@gmail.com>
Date: Fri, 8 May 2026 21:22:44 +0800
Subject: [PATCH 2/2] ENH: Update PythonQt to use forked repository

Switch PythonQt git repository from commontk/PythonQt to
hubutui/PythonQt and update revision tag to 41255c108e15.

Signed-off-by: Hu Butui <hot123tea123@gmail.com>
---
 CMakeExternals/PythonQt.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeExternals/PythonQt.cmake b/CMakeExternals/PythonQt.cmake
index 2aa00a96..fb28b997 100644
--- a/CMakeExternals/PythonQt.cmake
+++ b/CMakeExternals/PythonQt.cmake
@@ -91,7 +91,7 @@ if(NOT DEFINED PYTHONQT_INSTALL_DIR)
 
   ctkFunctionExtractOptimizedLibrary(PYTHON_LIBRARIES PYTHON_LIBRARY)
 
-  set(revision_tag 7ef4c5ee066b8ef1e7dc609b14071d504f59d4ba) # patched-v4.0.1-2026-03-16-5cd9b581f
+  set(revision_tag 41255c108e157760f3f349223de87fd5dd6baa36)
   if(${proj}_REVISION_TAG)
     set(revision_tag ${${proj}_REVISION_TAG})
   endif()
@@ -103,7 +103,7 @@ if(NOT DEFINED PYTHONQT_INSTALL_DIR)
     set(location_args GIT_REPOSITORY ${${proj}_GIT_REPOSITORY}
                       GIT_TAG ${revision_tag})
   else()
-    set(location_args GIT_REPOSITORY "https://github.com/commontk/PythonQt.git"
+    set(location_args GIT_REPOSITORY "https://github.com/hubutui/PythonQt.git"
                       GIT_TAG ${revision_tag})
   endif()
 
-- 
2.54.0

